<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info</class>
 <widget class="QMainWindow" name="info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ristorante Emilia</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>601</width>
      <height>291</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Benvenuti al &lt;/span&gt;&lt;span style=&quot; font-weight:700; color:#000000;&quot;&gt;Ristorante Da Emilia&lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;&gt;, dove la tradizione culinaria incontra il fascino del mare. Situato direttamente sulla costa, offriamo ai nostri ospiti un'esperienza unica tra sapori mediterranei e una vista mozzafiato sull'oceano. I nostri piatti sono preparati con ingredienti freschissimi, portando in tavola il meglio della cucina di mare e della terra. Che si tratti di un pranzo rilassante sotto il sole o di una cena romantica al tramonto, il nostro ambiente accogliente e il servizio attento faranno di ogni visita un momento indimenticabile.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'-webkit-standard'; font-size:medium; color:#000000;&quot;&gt;Oltre alla nostra ricca offerta gastronomica, organizziamo serate a tema, degustazioni di vini locali e musica dal vivo, creando un'atmosfera vivace e piacevole per ogni occasione. Disponiamo anche di una selezione di cocktail e bevande da gustare al bar, perfette per un aperitivo fronte mare. Siamo il luogo ideale per festeggiare eventi speciali come compleanni, matrimoni o cene aziendali, con la possibilità di riservare aree private per garantire un'esperienza personalizzata e memorabile.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple Chancery</family>
      <pointsize>36</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Informazioni</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Indietro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salva">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>0</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Salva</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
